checker/manual/annotating-libraries.tex

```diff
@@ -603,6 +609,17 @@ \subsection{Limitations\label{stub-limitations}}
   \begin{Verbatim}
           void init(@Nullable java.security.SecureRandom random);
    \end{Verbatim}
+\item
+   Annotations on types that are inner classes must written before the package on the
+   fully annotated type or before the inner class name. For example,
+
+\begin{Verbatim}
+        ProcessBuilder redirectError(@NonNull java.lang.ProcessBuilder.Redirect destination)
+\end{Verbatim}
+ or
+\begin{Verbatim}
+        ProcessBuilder redirectError(@NonNull Redirect destination)
+\end{Verbatim}
```

Comment:
Is it possible to write the parameter's type as "ProcessBuilder.Redirect"?  Showing that example, or explaining that it is not permitted to be written, would be helpful.